docs/content/direct/README.md

```diff
@@ -34,7 +34,7 @@ including OCM Klusterlet for the WECs.
 ## Latest stable release
 
 We do not have one that is proven very good yet.
-The latest release is [0.20.0](../../../../v0.20.0).
+The latest release is [0.21.0-rc1](../../../../v0.21.0-rc1).
```

Comment:
Since this is only an RC, maybe also retain mention of 0.20.0?

Reply:
Or maybe let's change to :
The latest stable release: 0.20.0
The latest development release candidate is: 0.21.0-rc1

Reply:
@MikeSpreitzer What do you think ?

Reply:
Since 0.20.0 is deliberately feature-incomplete, I would be clear about that.  How about something like the following?

> The first release using the new architecture is [0.20.0](../../../../v0.20.0); it is feture-incomplete.
> The latest release is [0.21.0-rc1](../../../../v0.21.0-rc1); it is also feature-incomplete.

Reply:
fixed